SPECS/pyproject-rpm-macros/pyproject-rpm-macros.spec

```diff
@@ -1,13 +1,23 @@
 Summary:        RPM macros for PEP 517 Python packages
 Name:           pyproject-rpm-macros
+
+%if 0%{?with_check}
+%bcond tests 1
+%else
+%bcond tests 0
+%endif
+# pytest-xdist and tox are not desired in RHEL
+#%%bcond pytest_xdist %%{undefined rhel}
+#%%bcond tox_tests %%{undefined rhel}
```

Comment:
I think we might as well replace all `%if 0%{with test}` with `%if 0%{with_check}`, since we regulate test vs non-test builds with the `with_check` macro.

And I think we could also remove the RHEL-specific comment below and any macros we're not setting like `pytest_xdist` or `tox_tests`.

Reply:
ok, I can do that, but wanted to minimize the changes from the fedora spec, where possible

Reply:
fixed